documentation/ci-fix.md

```diff
@@ -49,18 +49,31 @@ Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/st
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
 
 ## Breaking Change Check
-
+There are two kind of breaking change checks: same version breaking change check and cross version breaking change check. Both uses the OAD (OpenAPI Diff) tool to identify breaking changes between swaggers.
+- The same version breaking change check compares the existing version swaggers with updated swaggers of the same version, so that to identify if there are any updates break the version.
+- The cross version breaking change check usually applies to new API version scenario. It uses swaggers of the latest public version as the baseline, comparing swaggers of new version with the baseline, so that to identify if there are any updates lead to breaking experience cross versions.
+The latest public version can be:
+1. The latest stable version which released within 3 years.
+2. When there is no stable version released within 3 years, then the latest preview version which released within 1 years.  
```

Comment:
When there is no stable version, the latest preview version that is more than 1 year old.

Reply:
The real logic is: When there is no stable version(or latest stable version is over 3 year), Then latest preview version can not be break(no matter if over1 year.   maybe Jianye you can re-wording this part)

Reply:
Oh, I think I see what you're going for here. You're saying that it is OK to break if the latest stable is > 3 years old, right? If so, I think this is a bit dangerous and probably shouldn't be a rule. Azure's policy is that customers have to be given 3 years notice before **_forcing_** a breaking version on them. So, a version could be 10 years old but, if we haven't notified customers that the next version is breaking, then the breaking change is NOT ok. In other words, the trigger is notification (after approval); not duration of latest stable version. 

Also, what is more common is for a breaking change to be introduced within 3 years of the last stable and then, with approval & customer notification, this is allowed.

Reply:
yes, agree with you, Jeffery, As most common case is compare with stable version within 3 year. we can remove this condition(<= 3year) from tool. if so, for the case (> 3 year stable version), if there is breaking change, we will ask service team to request approval from you. :)

Reply:
@JeffreyRichter any concern about the above proposal by @akning-ms